packages/react/src/util/elementOverflow.ts

```diff
@@ -1,9 +1,47 @@
-export function elementOverflow(element: HTMLElement, container: HTMLElement) {
+// Default for `getContainer`, meant for use in suggestion menus. Traverses
+// ancestors of the passed element until a suggestion menu root is found.
+const defaultGetContainer = (element: HTMLElement) => {
+  let container: HTMLElement = element;
+
+  while (
+    !container.classList.contains("bn-suggestion-menu") &&
+    !container.classList.contains("bn-grid-suggestion-menu") &&
+    container.parentElement
+  ) {
+    container = container.parentElement;
+  }
+
+  if (
+    !container.classList.contains("bn-suggestion-menu") &&
+    !container.classList.contains("bn-grid-suggestion-menu")
+  ) {
+    return;
+  }
+
+  return container;
+};
+
+export function elementOverflow(
+  element: HTMLElement,
+  getContainer?: () => HTMLElement | undefined
+) {
+  let container = getContainer?.();
+  if (!container) {
+    container = defaultGetContainer(element);
```

Comment:
why not just say `getContainer = defaultGetContainer` in the function signature?